src/components/Calculator.js

```diff
@@ -1,35 +1,60 @@
-            <div className="btn number">7</div>
-            <div className="btn number">8</div>
-            <div className="btn number">9</div>
-            <div className="btn operation">X</div>
-            <div className="btn number">4</div>
-            <div className="btn number">5</div>
-            <div className="btn number">6</div>
-            <div className="btn operation">-</div>
-            <div className="btn number">1</div>
-            <div className="btn number">2</div>
-            <div className="btn number">3</div>
-            <div className="btn operation">+</div>
-            <div className="btn number zero">0</div>
-            <div className="btn number">.</div>
-            <div className="btn equal operation">=</div>
+            <button type="button" className="btn all-clear" onClick={() => this.pressedButton(obj, 'AC')}>AC</button>
+            <button type="button" className="btn last-clear" onClick={() => this.pressedButton(obj, '+/-')}>+/-</button>
+            <button type="button" className="btn " onClick={() => this.pressedButton(obj, '%')}>%</button>
+            <button type="button" className="btn operation" onClick={() => this.pressedButton(obj, '÷')}>/</button>
+            <button type="button" className="btn number" onClick={() => this.pressedButton(obj, '7')}>7</button>
+            <button type="button" className="btn number" onClick={() => this.pressedButton(obj, '8')}>8</button>
+            <button type="button" className="btn number" onClick={() => this.pressedButton(obj, '9')}>9</button>
+            <button type="button" className="btn operation" onClick={() => this.pressedButton(obj, 'x')}>X</button>
+            <button type="button" className="btn number" onClick={() => this.pressedButton(obj, '4')}>4</button>
+            <button type="button" className="btn number" onClick={() => this.pressedButton(obj, '5')}>5</button>
+            <button type="button" className="btn number" onClick={() => this.pressedButton(obj, '6')}>6</button>
+            <button type="button" className="btn operation" onClick={() => this.pressedButton(obj, '-')}>-</button>
+            <button type="button" className="btn number" onClick={() => this.pressedButton(obj, '1')}>1</button>
+            <button type="button" className="btn number" onClick={() => this.pressedButton(obj, '2')}>2</button>
+            <button type="button" className="btn number" onClick={() => this.pressedButton(obj, '3')}>3</button>
+            <button type="button" className="btn operation" onClick={() => this.pressedButton(obj, '+')}>+</button>
+            <button type="button" className="btn number zero" onClick={() => this.pressedButton(obj, '0')}>0</button>
+            <button type="button" className="btn number" onClick={() => this.pressedButton(obj, '.')}>.</button>
+            <button type="button" className="btn equal operation" onClick={() => this.pressedButton(obj, '=')}>=</button>
```

Comment:
- [ ] I would suggest practicing passing Props to child components of the Calculate.js component, you should create a button component then pass down the button operation along with the stickhandler to it.